docs/spec_draft.md

```diff
@@ -680,6 +681,48 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.if
+
+### Semantics
+
+Returns the result of `true_branch` if `pred` is `true`, `false_branch` if
+`pred` is `false`.
```

Comment:
Also, let's call out that only one branch is executed - this is material to the semantics of the op. E.g. "Only one branch is executed depending on the value of `pred`".

Reply:
Captured this in first line of two lines.